### PR TITLE
fix(bp-masthead): fix asset path

### PR DESCRIPTION
**Pull request by seaerchin**
dup of #366 for legacy sites

### CODE REVIEW

File: _includes/bp-masthead.html [hunk 1]
```diff
@@ -1,61 +1,85 @@
...
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  Trusted website<Text style="letter-spacing: -3px">s</Text>
+                </a>
+              </p>
             </div>
-            <div class="col banner-content-layout is-hidden" id="banner-content">
-                <div class="banner-content">
-                    <img src="../assets/img/government_building.svg" class="banner-icon">
```
Comment (seaerchin) [inline _includes/bp-masthead.html:33]: relative path

File: _includes/bp-masthead.html [hunk 2]
```diff
@@ -1,61 +1,85 @@
...
-                            websites <span class="avoidwrap">(e.g. go.gov.sg/open)</span>.
-                            <a href="https://go.gov.sg/trusted-sites" class="banner-content-text bp-masthead-button-link bp-masthead-button-text"
-                               target="_blank" rel="noreferrer">
-                                Trusted website<Text style="letter-spacing: -3px">s</Text>
-                            </a>
-                        </p>
-                    </div>
-                </div>
-                <div class="banner-content">
-                    <img src="../assets/img/lock.svg" class="banner-icon">
```
Comment (seaerchin) [inline _includes/bp-masthead.html:48]: relative path

File: _includes/bp-masthead.html [hunk 3]
```diff
@@ -1,61 +1,85 @@
...
-                    <div class="banner-content-text-container">
-                        <strong class="banner-content-title">Secure websites use HTTPS</strong>
-                        <p>
-                            Look for a
-                            <strong> lock </strong>(<img src="../assets/img/lock.svg" class="inline-banner-icon">)
```
Comment (seaerchin) [inline _includes/bp-masthead.html:53]: relative path

File: _includes/bp-masthead.html [hunk 4]
```diff
@@ -1,61 +1,85 @@
...
+          <div
+            class="masthead-divider is-hidden"
+            id="masthead-divider"
+            style="padding-left: -12px; padding-right: -12px"
+          ></div>
+        </div>
+        <div class="col banner-content-layout is-hidden" id="banner-content">
+          <div class="banner-content">
+            <img
+              src="/assets/img/government_building.svg"
```
Comment (seaerchin) [inline _includes/bp-masthead.html:43]: fixed here

File: _includes/bp-masthead.html [hunk 5]
```diff
@@ -1,61 +1,85 @@
...
-                        <p>
-                            Look for a
-                            <strong> lock </strong>(<img src="../assets/img/lock.svg" class="inline-banner-icon">)
-                            or https:// as an added precaution. Share sensitive information only on official, secure websites.
-                        </p>
-                    </div>
-                </div>
+          </div>
+          <div class="banner-content">
+            <img src="/assets/img/lock.svg" class="banner-icon" />
```
Comment (seaerchin) [inline _includes/bp-masthead.html:66]: fixed here

File: _includes/bp-masthead.html [hunk 6]
```diff
@@ -1,61 +1,85 @@
...
+            <div class="banner-content-text-container">
+              <strong class="banner-content-title"
+                >Secure websites use HTTPS</strong
+              >
+              <p>
+                Look for a
+                <strong> lock </strong>(<img
+                  src="/assets/img/lock.svg"
```
Comment (seaerchin) [inline _includes/bp-masthead.html:74]: fixed here

**Review by harishv7 — APPROVED** [review]

**Pull request merged**